### ISSUE TITLE
replace images with css

### ISSUE THREAD

**Issue by timoschilling** [activeadmin/activeadmin#3564]
we have some pixel/svg icons that should be replaced with font awesome icons after #3553 is merged
- [app/assets/images/active_admin/datepicker/datepicker-input-icon.png](https://github.com/activeadmin/activeadmin/tree/master/app/assets/images/active_admin/datepicker/datepicker-input-icon.png)
- ~~[app/assets/images/active_admin/datepicker/datepicker-next-link-icon.png](https://github.com/activeadmin/activeadmin/tree/master/app/assets/images/active_admin/datepicker/datepicker-next-link-icon.png)~~ #3644
- ~~[app/assets/images/active_admin/datepicker/datepicker-prev-link-icon.png](https://github.com/activeadmin/activeadmin/tree/master/app/assets/images/active_admin/datepicker/datepicker-prev-link-icon.png)~~ #3644
- ~~[app/assets/images/active_admin/nested_menu_arrow.gif](https://github.com/activeadmin/activeadmin/tree/master/app/assets/images/active_admin/nested_menu_arrow.gif)~~ ~~#3612~~ #3862
- ~~[app/assets/images/active_admin/nested_menu_arrow_dark.gif](https://github.com/activeadmin/activeadmin/tree/master/app/assets/images/active_admin/nested_menu_arrow_dark.gif)~~ ~~#3612~~ #3862
- [app/assets/images/active_admin/orderable.png](https://github.com/activeadmin/activeadmin/tree/master/app/assets/images/active_admin/orderable.png)

**Comment by javierjulio** [comment]
@timoschilling I've removed all these images and replaced with CSS based arrows as those are more than sufficient, this includes the date picker next/prev icons which my pull request takes care of. The CSS can be overridden too easily to use font based icons if preferred but I think what I have is a great default. I dropped the date picker icon because I couldn't find a suitable replacement and didn't think it was that valuable, a descriptive placeholder is far more important (e.g. indicating which field is the start date and the end date).
